Installing-Metacontroller-s390x.md

```diff
@@ -0,0 +1,73 @@
+# Installing Metacontroller on an s390x k8s cluster
+
+[Metacontroller](https://github.com/metacontroller/metacontroller) is not available for the s390x platform out of the box. This [PR](https://github.com/metacontroller/metacontroller/pull/771) adds support but in the meantime it is possible to deploy metacontroller on s390x using the following recipe:
+
+## Build Metacontroller
+
+Prequisites:
+
+- an s390x virtual server (e.g. a LinuxONE box)
```

Comment:
An s390x VSI is not required to build this component. The build can also be achieved by emulation compilation